docs/loki-deployment.md

```diff
@@ -0,0 +1,160 @@
+# Overview
+This page lists the 2 common Loki deployments, to help user to better understand Loki architecture.
+
+Loki consists several components as below (Quote from slides [here](https://grafana.com/blog/2018/12/12/loki-prometheus-inspired-open-source-logging-for-cloud-natives/)):
+![Loki Arch](loki_arch.png)
+
+|  Component | Intro |
+| --- | --- |
+| Distributor | use consistent hashing to assign a logstream to an ingester |
+| Ingester | gzip logs and append them. Index and data chunks go to different store |
+| Querier | look at the index to figure out which chunks match, and distributed grep through them. Talks to chunks and Ingester(get the recent data that has not been flushed) both |
+| Table-Manager | manage the chunk and index lifecycle |
+
+
+# Loki Deployment on Single Process
+To those who want to just have a try at Loki, deploying Loki on `Single Process` is the easiest way.
```

Comment:
Single Process Loki is not just for trying it out.

You can definitely run Loki in this mode in production and even horizontally scale it, just all components at once.

This might even be the easier approach to get it running in production